WooCommerce/Classes/ViewRelated/Blaze/BudgetSetting/BlazeBudgetSettingViewModel.swift

```diff
@@ -31,8 +34,13 @@ final class BlazeBudgetSettingViewModel: ObservableObject {
         return String.localizedStringWithFormat(Localization.dailyAmount, formattedAmount)
     }
 
+    var totalAmountTitle: String {
+        isEvergreen ? Localization.weeklySpend : Localization.totalSpend
+    }
+
     var totalAmountText: String {
-        let totalBudget = calculateTotalBudget(dailyBudget: dailyAmount, dayCount: dayCount)
+        let duration = isEvergreen ? 7 : dayCount
```

Comment:
nit: Can we make the `7` a constant to make it clear that we use it for a weekly budget? We use reuse it [here](https://github.com/woocommerce/woocommerce-ios/pull/13523/files#diff-273866c6580cadee3926c97eed92a6a5f4a15b45bb70e68d2514abd91dcabc35R412) as well.

Reply:
Good point! I added the changes in f80f7e47f309559742cea63cd5c8555d8b2bb0ae and 4d37b1911d4d869cf15066f35f17a93ae3b8f5c8.